try {
  ...
} catch (AnException e) {
  ExceptionCatcher.trap(e).process();
}